packages/google_sign_in/README.md

```diff
@@ -0,0 +1,84 @@
+# google_sign_in_tizen
+
+[![pub package](https://img.shields.io/pub/v/google_sign_in_tizen.svg)](https://pub.dev/packages/google_sign_in_tizen)
+
+The Tizen implementation of [`google_sign_in`](https://github.com/flutter/plugins/tree/master_archive/packages/google_sign_in/google_sign_in).
+
+## Usage
+
+This package is not an _endorsed_ implementation of `google_sign_in`. Therefore, you have to include `google_sign_in_tizen` alongside `google_sign_in` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  google_sign_in: ^5.4.0
+  google_sign_in_tizen: ^0.1.0
+```
+
+For detailed usage on how to use `google_sign_in`, see https://pub.dev/packages/google_sign_in#usage.
+
+## Tizen integration
+
+`google_sign_in_tizen` supports Google Sign-In for [TVs and Limited Input Devices](https://developers.google.com/identity/gsi/web/guides/devices), note that many API scopes are limited for these device types, see [supported scopes](https://developers.google.com/identity/protocols/oauth2/limited-input-device#allowedscopes).
+
+The plugin uses [OAuth 2.0 Device Authorization Grant](https://datatracker.ietf.org/doc/html/rfc8628) (aka device flow) which has a different authorization flow than [Authorization Code Grant with PKCE](https://datatracker.ietf.org/doc/html/rfc7636) used for endorsed platforms (Android, iOS, and web). While every step needed to authorize the app for "authorization code grant with PKCE" happens on a single device, "device flow" requires the user to use a secondary device (PC, tablet, or desktop) to finalize the authorization.
+
+To access Google Sign-In for Tizen, you'll need to make sure to [register your application for TVs and limited input devices](https://developers.google.com/identity/gsi/web/guides/devices). You also need to add some additional code to your app to fully integrate `google_sign_in` on Tizen (explained below).
```

Comment:
The wording is not clear. What does "To access Google Sign-In for Tizen" mean?

Also which information do you want the user to read from the linked page? What should the user do to "register their application"?

Reply:
How about:

To use `google_sign_in` and sign-in to your account, you first need create credentials for your app ("Client ID" and "Client Secret"). See [register your application for TVs and limited input devices](https://developers.google.com/identity/gsi/web/guides/devices#get_a_client_id_and_client_secret) on how to create them.

You also need to add some additional code to your app to fully integrate `google_sign_in` on Tizen (explained below).

Reply:
sign-in to your account → This is wrong. The "developer" creates credentials for their app and the "user" of the app uses their account to sign in.

Reply:
Ah, you're right.

To allow users to sign-in to their account using `google_sign_in`,